### PR TITLE
Update lezhin.com.xml

### PR DESCRIPTION
**Pull request by SaschaNaz**
Some hosts now has valid certifications, social.lezhin.com now supports HSTS, and there is a new host.

### CODE REVIEW

**Comment by J0WI** [comment]
Let's trigger Travis again since test didn't run here.

File: src/chrome/content/rules/lezhin.com.xml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 	link.lezhin.com
 	-->
 	
 	<target host="lezhin.com" />
 	<target host="www.lezhin.com" />
 	<target host="m.lezhin.com" />
+	<target host="desk.lezhin.com" />
+	<target host="cdn.lezhin.com" />
+	<target host="acdn.lezhin.com" />
+	<target host="ncdn.lezhin.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/lezhin.com.xml]: ggtoon.lezhin.com
social.lezhin.com

It seems that there are also mirrors like:
\d+cdn.lezhin.com
\d+www.lezhin.com

Comment (SaschaNaz) [inline src/chrome/content/rules/lezhin.com.xml]: 1. I think social.lezhin.com has HSTS header, should it still be added?
2. What does `\d+` mean?

Comment (SaschaNaz) [inline src/chrome/content/rules/lezhin.com.xml]: I'm also curious how you are getting additional hosts.

Comment (J0WI) [inline src/chrome/content/rules/lezhin.com.xml]: Yes, as long as hosts are not HSTS preloaded we include them in the rules to secure the initial requests. Also some users (including Tor Browsers) don't cache HSTS.
See https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md for a guide about finding subdomains.

**Comment by J0WI** [comment]
You can't use regex that way in a target. I would propose to just document this behavior since this mirrors are not really in use.

**Comment by SaschaNaz** [comment]
I tried adding some rules for them, do you want me to replace them with comments?

**Pull request merged**